poller.go

```diff
@@ -69,17 +72,17 @@ func (p *poller) manipulate(c *chunk) outcome {
 		}
 	}
 
-	if output.Status == nil {
+	if len(output.Status) == 0 {
 		c.err = &UnexpectedQueryError{c.queryID, c.stream.Text, errNilStatus()}
 		return finished
 	}
```

Comment:
This block doesn't seem needed anymore.

It was there before as purely defensive programming because SDK 1 over-used pointers everywhere, and dereferencing a `nil` pointer causes a panic.

Checking for empty string is not necessary since it'll hit the `default` block of the switch.

Note that empty string would have hit the `default` block *before* anyway, so this change apart from being redundant is also subtly changing the logic!